<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дифференциатор дискретных функций</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="functionInput">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите функцию...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="startInput">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите начальную границу...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="endInput">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите конечную границу...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stepInput">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>Введите шаг...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="switchButton">
      <property name="text">
       <string>Найти производную!</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Назад</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="visible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="outputLabel1">
      <property name="text">
       <string>Текстовое поле 1</string>
      </property>
      <property name="visible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="outputLabel2">
      <property name="text">
       <string>Текстовое поле 2</string>
      </property>
      <property name="visible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>Функция должна содержать переменную 'x'.
	   Допустимые операции: "+", "-", "*", "/", "^", "opposite", "factorial", "sin", "cos", "tan", "acos", "asin", "atan".</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
